tests/mockLogzioListener/listener.py

```diff
@@ -0,0 +1,91 @@
+# noinspection PyUnresolvedReferences
```

Comment:
you can use ListenerHandlerTest(unittest.TestCase):
https://docs.python.org/2/library/unittest.html
integrates nice with IDE,maybe jenkins